app/services/bus_logic/hwpx2pdf.py

```
@@ -0,0 +1,122 @@
+        except Exception as e:
+            print(f"  ❌ 변환 실패: {e}")
+            error_count += 1
+        
+        finally:
+            # 한글 프로그램 정리
+            if hwp:
+                try:
+                    hwp.Clear(1)  # 문서 닫기
+                    hwp.Quit()    # 한글 종료
+                    time.sleep(0.5)
+                except:
```

Ruff (0.15.1)

[error] Do not use bare `except`

(E722)